tests/phpunit/elementor/app/import-export/processes/test-revert.php

```diff
@@ -73,6 +73,8 @@ public function test_run__revert_all_one_imported_only() {
 		$this->assertEquals( $base_posts, $after_revert__posts );
 		$this->assertNotEquals( $after_import__posts, $after_revert__posts );
 
+		$this->assert_valid_revert_sessions_option();
```

Comment:
how is this test related to the usage?

Reply:
It's not.
I just added this test into the revert since in the usage i'm using this option and the revert was not covering it with tests.